schema.graphql

```diff
@@ -837,6 +839,21 @@ type IndexerQueryFeePaymentAggregation @entity {
   paymentSource: PaymentSource!
 }
 
+type Operator @entity {
+  "Joint address of all 3 parties involved (Operator, Indexer, DataService)"
+  id: ID!
```

Comment:
Would there be any advantages if we had one operator per address then tracked which (indexer, dataservice) combinations they have or dont have access to? From an outsider's perspective it seems more intuitive for querying the data, but im not sure if it's a better model for the subgraph code.

Reply:
Not sure I follow 🤔 

I think it's not super complex as it is (it makes sense that the ID is the joint ID of all 3 parties involved, so, the operator address, the indexer and the data service, or rather, the operator address and the provision "ID")